content/03-starting-oss-projects.md

```diff
@@ -227,21 +228,21 @@ The *README.md* and the license text file should be there for all repositories.
 
 To make sure that a certain set of health files is always created, there are different possibilities:
 
-* One possibility is to us template repositories. These are repositories that contain the required set of initial health files. A new repositories can be created/copied from this template repository and thus it contains already the required set of health files. Some Git provider (GitHub, for example) provide [specific means](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file) to created the required healthe files per default.
+* One possibility is to use template repositories. These are repositories that contain the required set of initial health files. A new repositories can be created/copied from this template repository and thus it contains already the required set of health files. Some Git provider (GitHub, for example) provide [specific means](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file) to create the required health files per default.
 
 * Another possibility is to create repositories with a tool. Such tools create repositories based on some input data via the API's of the Git provider (GitHub.com, GitLab.com, Bitbucket.org etc.). Thus, they can help that repositories are compliant to the company guidelines (contain the required health files **and** team structure, for example). Based on such tools self services for repository creation could be offered that allow development teams creating repositories themselves. Often, companies develop such tools for their specific needs. We (the authors of this document) do not know generic repository creation tools.
 
 ### Providing license and copyright information
 
-License and copyright information must be declared properly for an open source project. This is relevant ...
+License and copyright information must be declared properly for an open source project. This is important for consumers of the project as well as for contributors. Furthermore source code often gets copied from one project to another, this makes it mandatory that all files carry license and copyright information (Note that a statement like *For license conditions please check LICENSE.txt* is not suited)
```

Comment:
I think the "..." was meant to be continued by the bullet points below. This breaks with adding the sentence. I think it's good to add the sentence, though. So maybe we can convert the following two bullet points in an additional sentence: "License and copyright information has to be declared for the parts of the project you developed as well as for external components (i.e. code developed by external parties) that are part of your repositories."

Reply:
+1